doc/user/content/sql/create-table.md

```diff
@@ -70,6 +83,20 @@ connections. They are always created in the special `mz_temp` schema.
 Temporary tables may depend upon other temporary database objects, but non-temporary
 tables may not depend on temporary objects.
 
+### Memory usage
+
+Every table is backed by an [index](/overview/api-components/#indexes) that
+materializes all of the data in the table. Unlike sources, you cannot create an
+"unmaterialized" table. Therefore you must ensure that the data you store in
```

Comment:
Dangling participle ("sources" is being grammatically equated to "you" when it should be equated to "tables"). "Unlike sources, tables cannot be created in an unmaterialized state" or "You cannot create an unmaterialized table, as you can with sources" would be correct.

Reply:
Whoops, thanks, rewritten to "Unlike sources, tables cannot be 'unmaterialized'."